refinery/factory_boy/utils.py

```diff
@@ -115,3 +147,40 @@ def create_dataset_with_necessary_models(
         dataset.save()
 
     return dataset
+
+
+def _create_dataset_objects(dataset, is_isatab_based, latest_version):
+    for i in xrange(1, latest_version+1):
+        file_store_item_uuid = str(uuid_builtin.uuid4())
+        file_store_item = FileStoreItemFactory(
+            uuid=file_store_item_uuid,
+            source="http://www.example.com/test.{}".format(
+                "zip" if is_isatab_based else "csv"
+            )
+        )
+        investigation_uuid = str(uuid_builtin.uuid4())
+        investigation = InvestigationFactory(
+            uuid=investigation_uuid,
+            isarchive_file=file_store_item.uuid if is_isatab_based else None,
+            pre_isarchive_file=(
+                None if is_isatab_based else file_store_item.uuid
+            ),
+            identifier="{}: Investigation identifier".format(dataset),
+            description="{}: Investigation description".format(dataset),
+            title="{}: Investigation title".format(dataset)
+        )
+
+        study_uuid = str(uuid_builtin.uuid4())
+        study = StudyFactory(
+            uuid=study_uuid,
+            investigation=investigation,
+            identifier="{}: Study identifier".format(dataset),
+            description="{}: Study description".format(dataset),
+            title="{}: Study title".format(dataset)
+        )
+        InvestigationLinkFactory(
```

Comment:
Use the whole allowed width of the line

Reply:
@hackdna I don't think there is anything specific in our style guide about this, and this pattern has been in use by myself and others for awhile now. I'm not sure why all of a sudden its blocking for this PR. 

I'd be open to a style change if you wanted to write something up, but the olny thing close in the wiki is:
```
"If an opening parenthesis or square bracket is at the end of a line then the closing one should be on a line by itself"
```
Which this pattern doesn't violate.

Reply:
This formatting has been used throughout the existing codebase for years. It is also implied in PEP8 and it was never added to the style guide because it was never in question. I'm not sure why this deviation has happened.

Reply:
I think deviation has probably happened, because different developers have different styles, and we haven't explicitly enforced one way to do this. 

We could probably configure flake8 to do what we want, but I'd feel better about making a decision on this as a team.

Reply:
Yeah, let's decide and config flake8. I didn't realize there was a preferred style.